using-timescaledb/ingesting-data.md

```diff
@@ -28,10 +28,52 @@ TimescaleDB. Read more about how to set up Prometheus with TimescaleDB in our
 [Prometheus tutorial][prometheus-tutorial]. You can also consider using [Grafana][grafana]
 or [other visualization tools][other-viz-tools] to visualize your metrics.
 
+## PostgreSQL and TimescaleDB output plugin for Telegraf [](postgresql-and-timescaledb-output-plugin-for-telegraf)
+
+Telegraf is an agent that collects, processes, aggregates, and writes metrics. Since it is plugin-driven for both the 
+collection and the output of data, it is easily extendable. In fact, it already contains over 200 plugins for gathering and
+writing different types of data. 
+
+We wrote the PostgreSQL output plugin which also has the ability to send data to a TimescaleDB hypertable. Telegraf handles 
+batching, processing, and aggregating the data collected prior to inserting that data into TimescaleDB.
+
+<!-- -->
+>:WARNING: The [pull request][pull-request] is open and currently under review by the Telegraf developers, waiting to be 
+merged. To give users the opportunity to try this functionality, we built [downloadable binaries][downloadable-binaries] of
+Telegraf with our plugin already included. 
+
+The PostgreSQL plugin extends the ease of use users get from leveraging Telegraf by handling schema generation and
```

Comment:
hmm this is not entirely correct. not sure we wanna rewrite though. a "pure" narrow model usually has one value per row but with the plugin you will have multiple values per row unless you set fields as jsonb, the tag configuration is more about storing tags normalized/denormalized.

Reply:
We use this verbage elsewhere. A fix could be to say: However, you can specify that you want to store metrics and their tags in a normalized manner, with a separate metadata table and foreign keys. 

I think this is a little confusing though, no?